tests/python/unittest/test_profiler.py

```diff
@@ -232,6 +233,41 @@ def test_continuous_profile_and_instant_marker():
     print(debug_str)
     profiler.set_state('stop')
 
+def test_aggregate_stats_valid_json_return():
+    file_name = 'test_aggregate_stats_json_return.json'
+    enable_profiler(file_name, True, True, True)
+    test_profile_event(False)
+    debug_str = profiler.dumps(format = 'json')
+    assert(len(debug_str) > 0)
+    print(debug_str)
+    # if the format is wrong, an exception will be thrown
+    target_dict = json.loads(debug_str)
+    profiler.set_state('stop')
+
+def test_aggregate_stats_sorting():
+    sort_by_options = {'avg': "Avg", 'min': "Min", 'max': "Max", 'count': "Count"}
+    ascending_options = [False, True]
+    def check_ascending(lst, asc):
+        assert(lst == sorted(lst, reverse = not asc))
+
+    def check_sorting(str, sb, asc):
+        target_dict = json.loads(debug_str)
```

Comment:
How is this accessible here ? shouldn't it be str? or change parameter name to 'debug_str'